tomcat/src/main/java/nextstep/jwp/service/AuthService.java

```diff
@@ -14,31 +16,50 @@ public class AuthService {
     private static final Logger log = LoggerFactory.getLogger(Http11Processor.class);
 
     private final SessionManager sessionManager;
+    private final InMemoryUserRepository inMemoryUserRepository;
 
-    public AuthService(SessionManager sessionManager) {
+    public AuthService(SessionManager sessionManager, InMemoryUserRepository inMemoryUserRepository) {
         this.sessionManager = sessionManager;
+        this.inMemoryUserRepository = inMemoryUserRepository;
+    }
+
+    public boolean isLoggedIn(String sessionId) {
+        return Objects.nonNull(sessionManager.findSession(sessionId));
     }
 
     public String login(String account, String password) {
-        User user = InMemoryUserRepository.findByAccount(account)
-                .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 계정이거나 비밀번호가 틀렸습니다."));
-        if (!user.isSamePassword(password)) {
-            throw new IllegalArgumentException("존재하지 않는 계정이거나 비밀번호가 틀렸습니다.");
-        }
+        User user = inMemoryUserRepository.findByAccount(account)
+                .orElseThrow(() -> new AuthException("존재하지 않는 계정이거나 비밀번호가 틀렸습니다."));
+        validatePassword(password, user);
+
+        log.info(user.toString());
 
         Session session = new Session(UUID.randomUUID().toString());
         sessionManager.add(session);
-
         return session.getId();
     }
 
+    private void validatePassword(String password, User user) {
+        if (!user.isSamePassword(password)) {
+            throw new AuthException("존재하지 않는 계정이거나 비밀번호가 틀렸습니다.");
```

Comment:
```suggestion
            throw new AuthException("비밀번호가 틀렸습니다.");
```
윗 코멘트와 마찬가지로 자세히 표현할 수 있을 것 같아 제안했습니다!